src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -218,7 +220,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Common\Interop\Unix\" />
+    <Folder Condition="'$(TargetsUnix)' == 'true'" Include="Common\Interop\Unix\" />
```

Comment:
Dunno - since we're adding an ItemGroup conditioned to Unix, I threw it in there anyway. The GUI can ignore it or not - it makes sense from a readability perspective.